machida/main.pony

```diff
@@ -33,7 +33,7 @@ actor Main
     Machida.start_python()
 
     try
-      SignalHandler(ShutdownHandler, Sig.int())
+      SignalHandler(MachidaShutdownHandler, Sig.int())
 
```

Comment:
Should `Sig.term()` be added here also?

Reply:
Like @SeanTAllen says, this was a special case that we added to fix a specific problem. Unless there's a problem with the behavior there no reason to handle `Sig.term()`.